lib/common/exceptions/internal/forbidden.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
 module Common
   module Exceptions
-    # Forbidden - We may eventually want different variations on this with distinct MinorCodes
+    # Forbidden
     class Forbidden < BaseError
       def initialize(options = {})
         @detail = options[:detail]
+        @code = options[:code] || i18n_field(:code, {})
```

Comment:
So I'm not at all opposed to this, but this was the basic idea behind BackendServiceException refactor, to make it more flexible. Why not just define the custom Forbidden types in i18n and use that?